fastestimator/network.py

```diff
@@ -117,6 +117,12 @@ def _forward_batch(batch: MutableMapping[str, Any], state: Dict[str, Any], ops:
             if op.outputs:
                 write_outputs_by_op(op, batch, data)
 
+    def run_step(self, batch: Dict[str, Any]):
```

Comment:
run step is already defined in the baseNework

Reply:
no it's not

Reply:
oh, I guess I only fixed that in my xai branch. should have return typing:


    def run_step(self, batch: Dict[str, Any]) -> Tuple[Dict[str, Any], Dict[str, Any]]:
        # Batch, Predictions
        raise NotImplementedError

Reply:
fixed